codec/testdata/full.firelog.golden.json

```diff
@@ -0,0 +1,2 @@
+[
```

Comment:
It seems strange that it generated no blocks, the original file has many of them https://github.com/streamingfast/firehose-near/blob/develop/codec/testdata/deep-mind.dmlog.golden.json.

I imagine you `GOLDEN_UPDATE=true` while `DMLOG` was not fully renamed everywhere to `FIRE`